app/src/main/java/org/phenoapps/verify/LoaderDBActivity.java

```diff
@@ -105,7 +95,8 @@ protected void onCreate(Bundle savedInstanceState) {
 
         mFileUri = getIntent().getData();
 
-        int lastSlash = mFileUri.getPath().lastIndexOf('/');
+        assert mFileUri != null;
```

Comment:
Replace all new and old usages of assert with try catches and display errors to user when caught.